src/components/ChatEntry.js

```diff
@@ -1,22 +1,49 @@
 import React from 'react';
+import { useState } from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
+  const time = <TimeStamp timeStamp={props.timeStamp} />;
+  const nameLocalRemote = props.sender === 'Vladimir' ? 'local' : 'remote';
+
+  const classes = `chat-entry ${nameLocalRemote}`;
+
+  // Source: https://stackoverflow.com/questions/34521797/how-to-add-multiple-classes-to-a-reactjs-component
+  //Wave 3
+  const [like, setLike] = useState(false);
```

Comment:
This state variable is not needed.  When the heart button is clicked, it calls toggleLike on line 17, which will update the state variable chatsList in App, which will update the liked attribute of the object with the matching ID.  That object will get passed down to this component as a prop.  On line 21, you can use 'props.liked' to pick the correct display status.